pkg/acpi/bios.go

```diff
@@ -24,11 +24,14 @@ type BiosTable struct {
 // i.e. it goes straight to memory and gets them there. We optimistically
 // hope the bios has not stomped around in low memory messing around.
 func ReadBiosTables() (*BiosTable, error) {
-	r, err := GetRSDPEBDA()
+	r, err := GetRSDPEFI()
 	if err != nil {
-		r, err = GetRSDPMem()
+		r, err = GetRSDPEBDA()
```

Comment:
There is a slightly more idiomatic way to write this, 
r, err := x()
if ! err {
return r, nil
}

r, err := y()
if ! err {
return r, nil
}

and so on. 

Also, the way you are doing this, it may confuse people when they think the *only* is EBDA, not all of them (see line 33)

so: 
r, errors := GetRSDPEBDA()
if errors == nil {
return r, nil
}
r, err := GetRSDPMem()
if err != nil {
return r, nil
}
errors = errors.Join(errors, err)
etc.
If you can rewrite with this in mind, it would be an improvement, but I'll leave it up to you